terraform/services/config/templates/loris.ini.template

```diff
@@ -51,7 +51,7 @@ templates = 'wordpress,s3,prismic'
     url = 'https://wellcomecollection.files.wordpress.com/%s'
 
     [[s3]]
-    url = 'http://s3-eu-west-1.amazonaws.com/miro-images-public/%s'
+    url = 'http://s3-eu-west-1.amazonaws.com/wellcomecollection-miro-images-public/%s'
```

Comment:
Since we’ve never really exposed the `s3://` prefix publicly (it’s in our info.json's, but nobody looks at those), is it worth rethinking the name of this template?

Maybe call it `miro`? To distinguish from other collections we may store in S3 later?

Reply:
I think you're correct but as this change is already quite significant (and what you suggest is unnecessary to meet the goal of this PR) i'd prefer to do that in a later patch.